src/services/repository.ts

```diff
@@ -90,8 +90,8 @@ export default class Repository {
   }
 
   async #fetchLastGithubCommit() {
-    // eslint-disable-next-line max-len
-    const uri = `${this.#repository.commits_url.slice(0, this.#repository.commits_url.length - kCommitUrlPostfixLen)}?per_page=${kMaxCommitFetch}`;
+    const uri = `${this.#repository.commits_url
+      .slice(0, this.#repository.commits_url.length - kCommitUrlPostfixLen)}?per_page=${kMaxCommitFetch}`;
```

Comment:
I think we can use `URL` + `URL.searchParams` here:

```suggestion
    const url = new URL(this.#repository.commits_url.slice(0, this.#repository.commits_url.length - kCommitUrlPostfixLen));
    url.searchParams.set("per_page", kMaxCommitFetch.toString());
```
Note the `uri` -> `url` which is more accurate imo